### ISSUE TITLE
Build is currently broken

### ISSUE THREAD

**Issue by JasonGross** [mit-plv/fiat-crypto#85]
```
/usr/bin/time -f "src/CompleteEdwardsCurve/ExtendedCoordinates (real: %e, user: %U, sys: %S, mem: %M ko)" "coqc"  -q  -R "src" Crypto -R "Bedrock" Bedrock   src/CompleteEdwardsCurve/ExtendedCoordinates.v
File "./src/CompleteEdwardsCurve/ExtendedCoordinates.v", line 74, characters 85-89:
Error: Attempt to save an incomplete proof (in proof Equivalence_eq)
```

**Comment by andres-erbsen** [comment]
Looking into it (but worksforme).

**Comment by andres-erbsen** [comment]
This specific issue has been fixed, and 8.5 build passes.
